ftplugin/ruby.lua

```diff
@@ -1,6 +1,30 @@
-local cmd = vim.cmd
-cmd(
-  [[command! -nargs=? -complete=file RubyRun lua require("ruby_nvim.cmd").run(<f-args>)]])
-cmd([[command! RubyAlternate lua require("ruby_nvim.cmd").alternate()]])
-cmd([[command! -bang RubyTest lua require("ruby_nvim.cmd").test("<bang>")]])
-cmd([[command! -nargs=0 RubyBrowseGem lua require('ruby_nvim.cmd').browse_gem()]])
+local ruby_cmd = require("ruby_nvim.cmd")
+VIM_VERSION_MINOR_SUPPORTED = 7
+if vim.version().minor < VIM_VERSION_MINOR_SUPPORTED then
+  vim.notify("Sorry, but ruby.nvim doesn't support minors less than " .. VIM_VERSION_MINOR_SUPPORTED, "error")
+  return
+end
+
+vim.api.nvim_create_user_command("RubyRun", function(opts)
+  local arg
+  if opts.fargs ~= '' then
+    arg = opts.fargs[1]
+  else
+    arg = nil
+  end
+  ruby_cmd.run(arg)
```

Comment:
I have the feeling this whole function could be expressed as `ruby_cmd.run(opts.fargs[1] or nil)` — which is simpler and still easy to read IMHO.

Or even `ruby_cmd.run(opts.fargs[1])` if calling `ruby_cmd.run('')` and `ruby_cmd.run(nil)` has the same effect…

Reply:
I'll try, one minute

Reply:
Done